--- origsrc/clutter-gst-1.9.92/doc/reference/Makefile.am	2012-02-22 06:15:16.000000000 -0600
+++ src/clutter-gst-1.9.92/doc/reference/Makefile.am	2012-12-18 21:52:22.335131900 -0600
@@ -11,6 +11,7 @@ AUTOMAKE_OPTIONS = 1.6
 
 # The name of the module, e.g. 'glib'.
 DOC_MODULE=clutter-gst
+DOC_MODULE_VERSION=@CLUTTER_GST_MAJORMINOR@
 
 # The top-level SGML file. You can change this if you want to.
 DOC_MAIN_SGML_FILE=clutter-gst-docs.sgml
